src/components/com_tjdashboard/site/assets/js/tjDashboardUI.js

```diff
@@ -37,11 +38,15 @@ var TJDashboardUI = {
 					value.params = JSON.parse(value.params);
 				}
 
-				if(value.params.color!=undefined && value.params.color.length!=0){
+				if(value.params.color){
 					colorClass=value.params.color;
 				}
 
-				jQuery('<div class="col-xs-' +value.size+'"><div class="widget-data panel '+colorClass+'"><div class="widget-title panel-heading"><b>'+value.title+'</b></div><div data-dashboard-widget-id="'+value.dashboard_widget_id+'" id="dashboard-widget-'+value.dashboard_widget_id+'" class=""></div></div></div>').appendTo('.dashboard-widget-row-'+j);
+				if(value.params.icon){
```

Comment:
Same as above comment

Reply:
it is also defined. This changes told by Amol Sir.

Reply:
You have set the default value of `colorClass` and `icon` variables. I'm talking about the icon and color attributes in the config JSON.